apollo-compiler/src/main/kotlin/com/apollostack/compiler/FieldTypeSpecBuilder.kt

```diff
@@ -36,11 +36,19 @@ class FieldTypeSpecBuilder {
   /** Returns a list of types referenced by the inner fields in the provided fields */
   private fun innerTypes(fields: List<Field>, fragments: List<Fragment>): List<TypeSpec> =
       fields.filter(Field::isNonScalar)
-          .map { build(it.normalizedName(), it.fields!!, it.referencedFragments(fragments),
-              it.inlineFragments ?: emptyList()) }
+          .map { build(it.normalizedName(), it.fields!!, fragments, it.inlineFragments ?: emptyList()) }
```

Comment:
we shouldn't filter all fragments (that's why `it.referencedFragments(fragments)` removed) as they must propagated down to nested fields, as they can be used there